platform/src/main/java/platform/pageobjects/MyPage/Membership.java

```diff
@@ -0,0 +1,132 @@
+package platform.pageobjects.MyPage;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import utils.CommonFunctions;
+
+public class Membership {
+
+	List<String[]> testdata;
+	public WebDriver driver;
+	private static Logger log = LogManager.getLogger(Membership.class.getName());
+	int waitTime = 5;
+
+	public Membership(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+
+	@FindBy(xpath = "//a[@class='link js-not-link']")
+	public WebElement accountInformation;
+
+	@FindBy(xpath = "//a[@class='submenu__link'][contains(text(),'会員ステータス')]")
+	public WebElement membershipStatus;
+
+	@FindBy(xpath = "//div[@class='link-learn__content']")
+	public WebElement membershipButton;
+
+	@FindBy(xpath = "//button[@type='submit']")
+	public WebElement applyPrime;
+
+	@FindBy(xpath = "//a[@class='box-link__link box-link__link-blue']")
+	public WebElement primeBenefit1;
```

Comment:
Kindly include type of element with name whenever possible. for eg: primeBenefit1Link etc.

Reply:
I have added naming as suggested